src/Pact/Analyze/Check.hs

```diff
@@ -529,8 +529,8 @@ moduleModelDecl ModuleData{..} = do
   pure $ ModelDecl (HM.fromList propList) checkList
   where
     model = case _mdModule of
-      Pact.Module{Pact._mMeta=Pact.Meta _ m}            -> m
-      Pact.Interface{Pact._interfaceMeta=Pact.Meta _ m} -> m
+      Pact.MDModule Pact.Module{Pact._mMeta=Pact.Meta _ m}            -> m
```

Comment:
note to self: we might want `moduleDefMeta`

Reply:
done